internal/storage/driver/memory/binding.go

```diff
@@ -61,3 +61,15 @@ func (s *Binding) ListByInstanceID(instanceID string) ([]internal.Binding, error
 
 	return bindings, nil
 }
+
+func (s *Binding) Get(instanceID string, bindingID string) (*internal.Binding, error) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	binding := s.data[bindingID]
+	if binding.InstanceID == instanceID {
```

Comment:
Is it really possible? If the relation instance -> bindings is 1->n and no binding is shared by instances.

Reply:
I want to connect search with both bindingId and instanceId available in URL. Otherwise, it would be possible to load bindings from different instance than the one that instanceId refers to.

Reply:
We need to discuss it, I do not get it.

Reply:
Got it.